docs/tools-ides-ballerina-composer.md

```diff
@@ -30,81 +26,10 @@ As you develop new connectors, actions, and functions that you want to share wit
 
 ## Test framework
 
-When you write your code in Ballerina Composer, the syntax is checked for you as you write it, and you can use the Debug button to step through your program. You can also manually test a Ballerina file using the following command:
+When you write your code in aforementioned editors and IDEs, the syntax is checked for you as you write it, and you can use the Debug button to step through your program. You can also manually test a Ballerina file using the following command:
 
 ```
-./ballerina test <package_name>
+./ballerina test <module_name>
 ```
```

Comment:
Here the command given does not tally with the description. The description says the command is to test a Ballerina file but the command given is the way to run tests in a module.  We can correct the command to be:
```bash
'./ballerina test <balfile>'
```
Or else we can update the description to mention both the commands : testing a Ballerina file and a module as:
```bash
You can also manually test a Ballerina file or a module using the following commands:

./ballerina test <balfile>
./ballerina test <module_name>
```